nav2_mppi_controller/include/nav2_mppi_controller/optimizer.hpp

```diff
@@ -56,7 +50,7 @@ namespace mppi
  */
 class Optimizer
 {
-public:
+public:  
```

Comment:
Lint

Reply:
There's a few linting issues, I'm sure CI picks up on it once the merge conflicts are handled, or using ament_cpplint and ament_uncrustify locally